File: src/main/java/com/mhl/mycompanybackend/utils/ErrorMessages.java
package com.mhl.mycompanybackend.utils;

import com.mhl.mycompanybackend.pojo.MessageResponse;
import org.springframework.http.ResponseEntity;
import org.springframework.stereotype.Component;

@Component
public class ErrorMessages {
    public ResponseEntity<MessageResponse> badRequest(){
        return ResponseEntity.badRequest().body(new MessageResponse("Bad request"));
    }
}

File: src/main/java/com/mhl/mycompanybackend/utils/ErrorMessages.kt
package com.mhl.mycompanybackend.utils

import com.mhl.mycompanybackend.pojo.MessageResponse
import org.springframework.http.ResponseEntity
import org.springframework.stereotype.Component

@Component
class ErrorMessages {
    fun badRequest(): ResponseEntity<MessageResponse> {
        return ResponseEntity.badRequest().body(MessageResponse("Bad request"))
    }
}
